### PR TITLE
Make links and the serach bar more clear

### LINKED ISSUE #703: Make links more clear.
Use either [link](https://design.firefox.com/icons/viewer/#link) or [share windows](https://design.firefox.com/icons/viewer/#share%20windows) icon at the right of the link header (if link is available).

It might not be obvious to first time users, that there's a link to be used there.
(quick demo, icon should probably be more white, and smaller, and not aligned by hand)
image

### PR DESCRIPTION
**Pull request by AverageHelper**
Screen Shot 2021-08-08 at 6 48 16 PM

* Added a "Search" placeholder to the search area, with translations (to the best of the knowledge of DuckDuckGo's Translate feature)
* Added a link icon to link headers

Closes #703

### CODE REVIEW

File: _data/trans/pt_br.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "popular": "Popular",
     "pullrequest": "Faça um 'Pull Request' no GitHub",
     "reset": "Reset",
+    "search": "Procurar",
```
Comment (tupaschoal) [inline _data/trans/pt_br.json:34]:
```suggestion
    "search": "Pesquisar",
```

File: _data/trans/sr.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "popular": "Популарно",
     "pullrequest": "Форкујте пројекат на Гитхабу",
     "reset": "Reset",
+    "search": "Претраге",
```
Comment (tupaschoal) [inline _data/trans/sr.json:34]:
```suggestion
    "search": "Претрага",
```

File: _data/trans/tr.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "popular": "Popüler",
     "pullrequest": "GitHub'da Pull Request oluşturun",
     "reset": "Sıfırla",
+    "search": "Aramak",
```
Comment (tupaschoal) [inline _data/trans/tr.json:34]:
```suggestion
    "search": "Arama",
```

File: _data/trans/zh-cn.json [hunk 4]
```diff
@@ -31,6 +31,7 @@
     "popular": "Popular",
     "pullrequest": "Fork the project on GitHub",
     "reset": "Reset",
+    "search": "西尔克",
```
Comment (tupaschoal) [inline _data/trans/zh-cn.json:34]:
```suggestion
    "search": "搜索",
```

File: script/validate_json.rb [hunk 5]
```diff
@@ -27,7 +27,7 @@ module ExitCodes
                         "extensionp3", "extensionp4", "extensionp5", "extensionp6", "footercredits",
                         "guide", "guideeasy", "guideexplanations", "guidehard", "guideimpossible",
                         "guidemedium", "hideinfo", "jgmd", "mikerogers", "name", "noinfo", "noresults", "noresultshelp",
-                        "popular", "pullrequest", "reset", "sendmail", "showinfo", "tagline", "title", "twitter", "whatisthis",
+                        "popular", "pullrequest", "reset", "search", "sendmail", "showinfo", "tagline", "title", "twitter", "whatisthis",
                         "whatisthis1", "whatisthis2", "whatisthis3", "whatisthis4"]
```
Comment (tupaschoal) [inline script/validate_json.rb:31]:
```suggestion
                        "popular", "pullrequest", "reset", "search", "sendmail", "showinfo", "tagline", "title", "twitter",
                        "whatisthis", "whatisthis1", "whatisthis2", "whatisthis3", "whatisthis4"]
```

**Review by tupaschoal — COMMENTED** [review]
Very nice improvements, thanks for coming up with them! The link icons on the titles will really help people know these can be clickable.

Aside from `pt_br` which is my main language, I double checked all of the translations on Google Translate and most of them seemed fine, except a few, so I suggested what Google came up with.

**Comment by AverageHelper** [comment]
TIL I can batch suggested changes together. Sorry about the commit spam 😅

**Comment by tupaschoal** [comment]
Thanks for this!

**Pull request merged**